src/coord/src/coord.rs

```diff
@@ -1552,6 +1552,9 @@ impl Coordinator {
                 self.ship_dataflow(df).await?;
             }
             self.maybe_begin_caching(source_id, caching_enabled).await;
+            let frontiers =
+                self.new_frontiers(source_id, Some(0), self.logical_compaction_window_ms);
+            self.sources.insert(source_id, frontiers);
```

Comment:
Why was it ok that we didn't have this before?

Update: I see, this is because `determine_frontier` needs it to return 0 when `least_valid_since` is called.